src/_includes/layout.njk

```diff
@@ -13,6 +13,14 @@
   <link rel="shortcut icon" href="/icons/favicon.ico">
   <meta name="msapplication-TileColor" content="#da532c">
   <meta name="msapplication-config" content="/icons/browserconfig.xml">
+  <meta property="og:title" content="{% if title != '' %}{{ title }}{% else %}{{ app.title }}{% endif %}">
+  <meta property="og:description" content="{% if description %}{{ description }}{% elif content %}{{ content | striptags | truncate(200) }}{% else %}{{ app.description }}{% endif %}">
+  <meta property="og:url" content="{{ page.url | prepend(app.url) }}">
+  <meta property="og:type" content="website">
+  <meta property="og:image" content="{{ app.url }}/og-image.png">
+  <meta name="twitter:creator" content="{% if creator %}@{{ creator }}{% else %}@FEFoundations{% endif %}">
+  <meta name="twitter:site" content="@FEFoundations">
+  <meta name="twitter:image" property="og:image" content="{{ app.url }}/og-image.png" />
```

Comment:
Lets update this to `{{ app.url }}/icons/apple-touch-icon.png` since `og-image.png` does not exist.